exporters/otlp/internal/transform/attribute_test.go

```diff
@@ -146,3 +148,147 @@ func TestAttributes(t *testing.T) {
 		}
 	}
 }
+
+func TestArrayAttributes(t *testing.T) {
+	// Array KeyValue supports only arrays of primitive types:
+	// "bool", "int", "int32", "int64",
+	// "float32", "float64", "string",
+	// "uint", "uint32", "uint64"
+	for _, test := range []attributeTest{
+		{nil, nil},
+		{
+			[]kv.KeyValue{
+				kv.Array("bool array to bool array", []bool{true, false}),
+				kv.Array("int array to int64 array", []int{1, 2, 3}),
+				kv.Array("uint array to int64 array", []uint{1, 2, 3}),
+				kv.Array("int32 array to int64 array", []int32{1, 2, 3}),
+				kv.Array("uint32 array to int64 array", []uint32{1, 2, 3}),
+				kv.Array("int64 array to int64 array", []int64{1, 2, 3}),
+				kv.Array("uint64 array to int64 array", []uint64{1, 2, 3}),
+				kv.Array("float32 array to double array", []float32{1.11, 2.22, 3.33}),
+				kv.Array("float64 array to double array", []float64{1.11, 2.22, 3.33}),
+				kv.Array("string array to string array", []string{"foo", "bar", "baz"}),
+			},
+			[]*commonpb.KeyValue{
+				newOtelBoolArray("bool array to bool array", []bool{true, false}),
```

Comment:
Since creating arrays is a bit more cumbersome that the simple values from the previous test, I though it might be better to have separate methods for this, for the sake of readability.